src/Agent/MsiInstaller/Installer/Product.wxs

```diff
@@ -215,9 +215,12 @@ SPDX-License-Identifier: Apache-2.0
 		<!-- Main Product -->
 		<ComponentGroup Id="ProductComponents" Directory="NETAGENTFOLDER">
 			<!-- Text -->
-			<Component Id="LicenseComponent" Guid="{47114807-F20A-478D-9F4F-4B3D6CF195BE}">
-				<File Id="LicenseFile" Name="License.txt" KeyPath="yes" Source="$(var.AgentSourcePath)\Miscellaneous\License.txt" />
-			</Component>
+      <Component Id="LicenseComponent" Guid="{47114807-F20A-478D-9F4F-4B3D6CF195BE}">
+        <File Id="LicenseFile" Name="LICENSE.txt" KeyPath="yes" Source="$(var.RootSourcePath)\licenses\LICENSE.txt" />
```

Comment:
Can we delete the license.txt in Miscellaneous then?

Reply:
Sure!